### DISCUSSION TITLE
OG Xbox internal install assistance.

### DISCUSSION THREAD

**Discussion by Zonark** [darthcloud/BlueRetro#1032] [Q&A]
I am attempting to perform an internal install on a OG Xbox, however I haven’t seen anyone else do this. Before I get into it is it even possible? I saw it has basically a usb connection with Data+ and Data- lines. Wiring to a esp32 wroom 32e.

**Comment by konwektor** [comment]
BR doesnt support OGXbox.
What fork are you using?
netham45 made fork to work with ogx360 by Ryzee119 - arduino boards 
netham45: [https://github.com/netham45/BlueRetro ](https://github.com/netham45/BlueRetro)? BR on ESP32chip, plus ogx360 on arduino
those are forks of netham45 fork :
jackryner: [https://github.com/jackrayner/BlueRetro](https://github.com/jackrayner/BlueRetro)? nice and clean IMHO
mine : [https://github.com/konwektor/BlueRetro](https://github.com/konwektor/BlueRetro)? couple different " in develop" branches - not finished internal , coz of my low programming skills.
or maybe davidx version [https://www.youtube.com/watch?v=r4PG4MqMDyg](https://www.youtube.com/watch?v=r4PG4MqMDyg)? in spanish????
